### PR TITLE
Schema as Markdown.

### PR DESCRIPTION
**Pull request by dkorolev**
Hi @mzhurovich,

After https://github.com/C5T/Current/pull/470, this PR enables Markdown schema export.

Thanks,
Dima

### CODE REVIEW

File: TypeSystem/Schema/schema.h [hunk 1]
```diff
@@ -389,6 +393,115 @@ struct LanguageSyntaxImpl<Language::FSharp> {
...
+        return "UNKNOWN_TYPE_" + current::ToString(type_id);  // LCOV_EXCL_LINE
+      } else {
+        struct MarkdownTypeNamePrinter {
+          const FullSchemaPrinter& self_;
+          std::ostringstream& oss_;
+
+          MarkdownTypeNamePrinter(const FullSchemaPrinter& self, std::ostringstream& oss)
+              : self_(self), oss_(oss) {}
+
+          // `operator()(...)`-s of this block print F# type name only, without the expansion.
```
Comment (mzhurovich) [inline TypeSystem/Schema/schema.h]: Update comment? `F#` is definitely from some other place :)

File: TypeSystem/Schema/schema.h [hunk 2]
```diff
@@ -389,6 +393,115 @@ struct LanguageSyntaxImpl<Language::FSharp> {
...
+    void operator()(const ReflectedType_Enum&) const {}
+    void operator()(const ReflectedType_Vector&) const {}
+    void operator()(const ReflectedType_Pair&) const {}
+    void operator()(const ReflectedType_Map&) const {}
+    void operator()(const ReflectedType_Optional&) const {}
+    void operator()(const ReflectedType_Variant&) const {
+      // No need to define `Variant` types explicitly in Markdown format.
+    }
+
+    // When dumping a `CURRENT_STRUCT` as an F# record, since inheritance is not supported by Newtonsoft.JSON,
```
Comment (mzhurovich) [inline TypeSystem/Schema/schema.h]: And here.

File: TypeSystem/Schema/schema.h (hunk 1 above)
Comment (dkorolev) [inline TypeSystem/Schema/schema.h]: :+1: 3738969

File: TypeSystem/Schema/schema.h (hunk 2 above)
Comment (dkorolev) [inline TypeSystem/Schema/schema.h]: :+1: 3738969

**Pull request merged**